pretix_eth/network/networks.py

```diff
@@ -25,6 +65,36 @@ class L1(INetwork):
     """ Implementation for Ethereum L1 """
     def __init__(self):
         super(L1, self).__init__("L1")
+        self.DAI_ADDRESS = '0x6b175474e89094c44da98b954eedeac495271d0f'
```

Comment:
Quick python note here: you don't have to use constructor `__init__` to assign "instance variables".  It will be bit more readable and "pythonic" if you do:

```python
class Rinkeby(INetwork):
    """ Implementation for Rinkeby Testnet """
    DAI_ADDRESS = '0x6b175474e89094c44da98b954eedeac495271d0f'

    def payment_instructions(self):
        ...
```

Of course there is a time when you wanna use real instance properties and this is the way of doing it, but in you case class variables should be enough, I believe!

Reply:
Cool! Will do!